src/app/slash_plugins/addclasschans.plugin.ts

```diff
@@ -169,54 +165,23 @@ const promptUser = async (
   const response =
     'making channels:\n```\n' +
     classes.map((v) => `${v.category}#${v.code} -- ${v.name}`).join('\n') +
-    '\n```\n respond CONFIRM or CANCEL';
+    '\n```';
 
   const messages = Util.splitMessage(response, { char: '\n', prepend: '```', append: '```' });
   await Promise.all(messages.map((m) => interaction.channel?.send({ content: m })));
 
-  const msg = (await interaction.reply({
-    content: 'Confirm or cancel',
-    components: [getConfirmCancelRow()],
-    fetchReply: true,
-  })) as Message;
-
-  const collector = msg.createMessageComponentCollector({
-    componentType: 'BUTTON',
-    time: ms('5m'),
-  });
-
-  collector.on('collect', async (btnInteraction) => {
-    await btnInteraction.deferUpdate();
-    if (btnInteraction.customId === 'confirm') {
-      await proceedToAddClasses(container, classes);
-      await btnInteraction.followUp({ content: 'Done' });
-    } else if (btnInteraction.customId === 'cancel') {
-      await proceedToCancel(interaction);
-      await btnInteraction.followUp({ content: 'Canceled' });
-    }
+  const onConfirm = async (btnInteraction: ButtonInteraction) => {
+    await btnInteraction.followUp('Creating channels...');
+    await proceedToAddClasses(container, classes);
```

Comment:
These can be done concurrently.
```suggestion
    await Promise.all([
        btnInteraction.followUp('Creating channels...'),
        proceedToAddClasses(container, classes),
    ]);
```